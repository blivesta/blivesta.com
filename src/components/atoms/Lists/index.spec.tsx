import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import List from '.';

describe('List', () => {
  it('snapshot', () => {
    const tree = shallow(
      <List
        items={[
          {
            title: 'foo',
            linkUrl: 'https://blivesta.com',
          },
          {
            title: 'bar',
          },
          {
            title: 'baz',
          },
        ]}
      />,
    );
    expect(toJson(tree)).toMatchSnapshot();
  });
});
