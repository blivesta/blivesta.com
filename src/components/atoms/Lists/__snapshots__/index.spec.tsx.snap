// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List snapshot 1`] = `
<styled.ul
  items={
    Array [
      Object {
        "linkUrl": "https://blivesta.com",
        "title": "foo",
      },
      Object {
        "title": "bar",
      },
      Object {
        "title": "baz",
      },
    ]
  }
>
  <styled.li
    key="0"
  >
    <styled.a
      href="https://blivesta.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      foo
    </styled.a>
  </styled.li>
  <styled.li
    key="1"
  >
    bar
  </styled.li>
  <styled.li
    key="2"
  >
    baz
  </styled.li>
</styled.ul>
`;
